pkg/synopsysctl/cmd_destroy.go

```
@@ -74,11 +79,12 @@ var destroyCmd = &cobra.Command{
 }
 
 func destroy(namespace string) error {
-	log.Infof("destroying Synopsys Operator in namespace '%s'...", namespace)
 	crds := []string{}
+	var isExist bool
+	// get the synopsys operator configmap to get the crd names and cluster scope
 	cm, err := util.GetConfigMap(kubeClient, namespace, "synopsys-operator")
 	if err != nil {
-		log.Errorf("unable to find the 'synopsy-operator' config map in namespace '%s' due to %+v", namespace, err)
+		log.Errorf("error getting the config map in namespace '%s' due to %+v", namespace, err)
```

return an error instead of logging

---

done